### PR TITLE
Add Catalan translation

### PR DESCRIPTION
**Pull request by fitojb**

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
Thanks!

File: po/ca.po [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+msgstr "Surt"
+
+#: data/resources/about.ui:10
+msgid "Copyright 2020 Rafael Mardojai CM"
+msgstr "Drets d’autor © 2020 Rafael Mardojai C. M."
+
+#. Replace me with your names
+#: data/resources/about.ui:14
+msgctxt "Add your names here, do not remove previous names!"
+msgid "translator1, translator2, ..."
```
Comment (fitojb) [inline po/ca.po:68]: This string should be GTK’s standard `translation-credits`, by the way @rafaelmardojai

Comment (rafaelmardojai) [reply po/ca.po:68]: What do you mean? `translation-credits` instead of `translator1, translator2, ...`?

Comment (fitojb) [reply po/ca.po:68]: Correct

**Pull request merged**